### PR TITLE
Changed latin "a" to cyrillic "а"

### PR DESCRIPTION
**Pull request by mingalevme**
Faker\Provider\ru_RU\Person uses suffix "a" to make a female last name from male, but letter is a latin "a" (ASCII #97) and not cyrillic "a" (ASCII #208), so the final last name is "some cyrillic symbols" + "latin a", commit fixes that, now female last name forms correctly: cyrillic male last name + cyrillic "a".

### CODE REVIEW

**Review by sanmon1985 — APPROVED** [review]

File: src/Faker/ORM/Doctrine/Populator.php [hunk 1]
```diff
@@ -98,11 +98,9 @@ public function execute($entityManager = null)
                 );
                 if (count($insertedEntities) % $this->batchSize === 0) {
                     $entityManager->flush();
-                    $entityManager->clear($class);
```
Comment (pimjansen) [inline src/Faker/ORM/Doctrine/Populator.php:101]: This is totally unrelated?

File: src/Faker/ORM/Doctrine/Populator.php [hunk 2]
```diff
@@ -98,11 +98,9 @@ public function execute($entityManager = null)
...
                 }
             }
             $entityManager->flush();
-            $entityManager->clear($class);
```
Comment (pimjansen) [inline src/Faker/ORM/Doctrine/Populator.php:105]: Unrelated?

**Review by pimjansen — CHANGES_REQUESTED** [review]
See my comments

File: src/Faker/ORM/Doctrine/Populator.php (hunk 1 above)
Comment (mingalevme) [reply src/Faker/ORM/Doctrine/Populator.php:101]: This is from already merged commit (https://github.com/fzaninotto/Faker/commit/5337ce5261b393937bd7aaebb21020159756a152), just because I branched the repo before.

**Comment by localheinz** [comment]
Closing in favour of #2056.
